docs/spec.md

```diff
@@ -1451,6 +1451,8 @@ where `min_val = rank(min) == 0 ? min : min[i0, ..., iR-1]`,
 // %result: [5, 13, 20]
```

Comment:
Thank you for addressing the comments! Please also update the PR description to follow the testing guidelines, e.g. similarly to #1080.

Reply:
Ah thanks for the reminder! Added, and will do so for the rest of my PRs.